src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --bluecolor: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Helvetica", sans-serif;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+  box-sizing: border-box;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  padding-top: 23px;
+  padding-bottom: 22px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: normal;
+  text-align: center;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  &:hover {
+    color: var(--bluecolor);
+  }
+  &:active {
+    color: var(--bluecolor);
+    border-bottom: var(--bluecolor) solid 4px;
+    border-radius: 1px;
+  }
+}
+
+.is-active {
+  color: var(--bluecolor);
+  border-bottom: var(--bluecolor) solid 4px;
+}
+
+.container {
+  display: flex;
+  padding-left: 50px;
+  align-items: center;
+}
+
+[data-qa="hover"]:hover {
+  color: var(--bluecolor);
+}
+
+.nav__list > :not(:first-child) {
+  margin-left: 20px;
+}
+
+.container > :first-child {
+  padding-top: 5px;
 }
```

Comment:
that is not the correct space from the top for the logo. 
Also you should not style the logo with `:first-child` pseudo-class. This way you make your code less predictable. `:first-child` is great for lists. flex elements, etc. but not for unrelated layout elements.